homeassistant/components/rympro/sensor.py

```diff
@@ -25,41 +40,40 @@ async def async_setup_entry(
     """Set up sensors for device."""
     coordinator: RymProDataUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id]
     async_add_entities(
-        RymProSensor(coordinator, meter_id, meter["read"], config_entry.entry_id)
+        RymProSensor(coordinator, meter_id, description, config_entry.entry_id)
         for meter_id, meter in coordinator.data.items()
+        for description in SENSOR_DESCRIPTIONS
     )
 
 
 class RymProSensor(CoordinatorEntity[RymProDataUpdateCoordinator], SensorEntity):
     """Sensor for RymPro meters."""
 
     _attr_has_entity_name = True
-    _attr_translation_key = "total_consumption"
     _attr_device_class = SensorDeviceClass.WATER
     _attr_native_unit_of_measurement = UnitOfVolume.CUBIC_METERS
-    _attr_state_class = SensorStateClass.TOTAL_INCREASING
 
     def __init__(
         self,
         coordinator: RymProDataUpdateCoordinator,
         meter_id: int,
-        last_read: int,
+        description: SensorEntityDescription,
         entry_id: str,
     ) -> None:
         """Initialize sensor."""
         super().__init__(coordinator)
         self._meter_id = meter_id
         unique_id = f"{entry_id}_{meter_id}"
-        self._attr_unique_id = f"{unique_id}_total_consumption"
+        self._attr_unique_id = f"{unique_id}_{description.translation_key}"
```

Comment:
We should not use the translation key for it. The translation key should only be used for translations.

```suggestion
        self._attr_unique_id = f"{unique_id}_{description.key}"
```